.env.example

```
@@ -4,3 +4,5 @@ CONVEX_DEPLOYMENT=
 VITE_CONVEX_URL=
 
 CONVEX_DEPLOY_KEY=
+
+DTU_BASE_URL=https://kurser.dtu.dk
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)